src/database/queries/sessionAttendees.js

```diff
@@ -52,28 +74,53 @@ exports.unEnrollFromSession = async (sessionId, userId) => {
 	}
 }
 
-exports.findAll = async (filter, options = {}) => {
+exports.unEnrollFromSessionWithValidation = async (sessionId, userId, tenantCode) => {
 	try {
+		const { sequelize } = SessionAttendee
+		const deleteQuery = `
```

Comment:
@sumanvpacewisdom insted of this we can use seqalize funtion, as we can use join now